src/index.ts

```diff
@@ -1,14 +1,22 @@
 import { KadDHT as SingleKadDHT } from './kad-dht.js'
 import { DualKadDHT } from './dual-kad-dht.js'
+import drain from 'it-drain'
+import { CodeError } from '@libp2p/interfaces/errors'
+import type { DHT, DualDHT, Selectors, Validators } from '@libp2p/interface-dht'
+import { ContentRouting, contentRouting } from '@libp2p/interface-content-routing'
+import type { CID } from 'multiformats/cid'
+import type { AbortOptions } from '@libp2p/interfaces'
+import type { PeerInfo } from '@libp2p/interface-peer-info'
 import type { ProvidersInit } from './providers.js'
-import type { Selectors, Validators } from '@libp2p/interface-dht'
 import type { Registrar } from '@libp2p/interface-registrar'
 import type { AddressManager } from '@libp2p/interface-address-manager'
 import type { PeerStore } from '@libp2p/interface-peer-store'
 import type { ConnectionManager } from '@libp2p/interface-connection-manager'
 import type { Metrics } from '@libp2p/interface-metrics'
-import type { PeerId } from '@libp2p/interface-peer-id'
 import type { Datastore } from 'interface-datastore'
+import { PeerRouting, peerRouting } from '@libp2p/interface-peer-routing'
+import { PeerDiscovery, peerDiscovery } from '@libp2p/interface-peer-discovery'
```

Comment:
Shouldn't this be imported as `@libp2p/peer-discovery` based on the [interface](https://github.com/libp2p/js-libp2p-interfaces/blob/master/packages/interface-peer-discovery/src/index.ts#LL12C1-L12C1) @achingbrain ?

Reply:
Ah, no - that’s a typo in the comment. Got to get that TS comment verification stuff in!